### ISSUE TITLE
A point against `LocalizableString`: redundancy

### ISSUE THREAD

**Issue by pchampin** [w3c/rdf-dir-literal#2]
While my first idea was indeed to introduce a new datatype (as the least invasive solution). But I came to think that it would be a very bad solution. It would recreate the "plain litteral / `xsd:string`" dilemma that plagued RDF 1.0 . We will have a mix of `LocalizedString`s and `langString`s, creating a mess for data consumers.

By avoiding to break standards and implementations, we may break the data ecosystem, which is much worse.

**Comment by iherman** [comment]
True. But it is not impossible that, on long term, an RDF WG working on RDF 2.0 could make this redundancy disappear, just as RDF 1.1 equates plain literal and `xsd:string`. So `LocalizableString` could be seen as a temporary solution as far as the core RDF model is concerned, but it would allow to deploy [updated serialization formats](https://w3c.github.io/rdf-dir-literal/#serializations-0), including SPARQ syntax. Ie, for most of the users the issue would become transparent...

**Comment by pchampin** [comment]
Equating plain literals with `xsd:string` worked without changing the concrete syntaxes. That's what made the transition so smooth -- even unnoticeable, for some.

`LocalizableString` would introduce a new way to write literals, distinct from `langString`. This will be much harder to mend, IMHO.
